### ISSUE TITLE
inhand sprites  don't update/update inversely

### ISSUE THREAD

**Issue by Pockets-byte** [unitystation/unitystation#6684]
Linux - staging

Currently the Energy sword, and a couple of other items I'm not thinking of right now, have issues with their sprites
the item appears to remain on the ground when toggled
as well as the item's state in-hand appearing inverted to it's actuall state
![Screenshot at 2021-06-04 19-25-58](https://user-images.githubusercontent.com/55299415/120871837-d3781680-c56a-11eb-84a2-34fc1507cbb8.png)

**Comment by NoooneyDude** [comment]
I had used a cheap hack to "fix" the sprite inversion issue. Looks like that was reverted.

Edit: https://github.com/unitystation/unitystation/pull/6256

**Comment by pyrosshade** [comment]
had noticed this with the null-rod as well

**Comment by corp-0** [comment]
> I had used a cheap hack to "fix" the sprite inversion issue. Looks like that was reverted.
> 
> Edit: #6256

was it reverted or overwritten by other pr?

**Comment by corp-0** [comment]
https://github.com/unitystation/unitystation/compare/unitystation:develop...Aranclanos:toyswordlight

Seems like files are still the same as on that PR.

**Comment by Fogapod** [comment]
related? https://github.com/unitystation/unitystation/issues/5508

**Comment by corp-0** [comment]
somewhat related but not exactly. This issue says the state sprite is inverted while the other issue is specifically saying the sprite from null rod is not updating transformation until it is picked up again.

**Comment by corp-0** [comment]
I could reproduce. 

Steps:
- Spawn energy sword and pick it up **as a client**
- Turn it on. Sprite is inverted, you still get the turning on sound and light effect
- Turn it off. Same as above but inverted.
- Turn it on again and drop it. The sprite will be correct.
- Pick it up, you will leave a ghost.

Important, to reproduce you must pick it up **as a client**

**Comment by NoooneyDude** [comment]
> [unitystation:develop...Aranclanos:toyswordlight](https://github.com/unitystation/unitystation/compare/unitystation:develop...Aranclanos:toyswordlight)
> 
> Seems like files are still the same as on that PR.

No, I mean to say, that PR removed my hack and reintroduced the issue. Though the hack was pretty bad too...
